chunky/src/java/se/llbit/chunky/ui/render/AdvancedTab.java

```diff
@@ -105,6 +106,19 @@ public AdvancedTab() throws IOException {
       PersistentSettings.setNumRenderThreads(value);
       renderControls.showPopup("This change takes effect after restarting Chunky.", renderThreads);
     });
+
+    octreeImplementation.getItems().addAll(Octree.ImplementationEnum.values());
+    octreeImplementation.getSelectionModel().selectedItemProperty()
+      .addListener((observable, oldvalue, newvalue) -> {
+        scene.setOctreeImplementation(newvalue);
+        PersistentSettings.setOctreeImplementation(newvalue.ordinal());
```

Comment:
Shouldn't the octree also be changed here? Or is it sufficient to do that on next chunk load? If so, maybe add a note that the new octree is used when reloading the chunks.

Reply:
Changing the octree could potentially be a heavy operation so i want the user to manually reload chunks (like when changing y min/max). Note added in the tooltip